### PR TITLE
Prevent /usr/local/bin/hyper overwriting

### LINKED ISSUE #2884: CLI: Linking must not replace existing file
A user can already have a legitimate file/link `/usr/local/bin/hyper`.
https://hyper.sh/

A notification should be issued instead of overwriting.

### PR DESCRIPTION
**Pull request by chabou**
Silently fail installation at startup.
Add a menu item to manually install it with a notification feedback.
Fix plugin update menu item.

Fixes #2884

### CODE REVIEW

File: app/menus/menus/plugins.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
-module.exports = (commands, update) => {
+module.exports = (commands, execCommand) => {
```
Comment (albinekb) [inline app/menus/menus/plugins.js]: Nice!

File: app/utils/cli-install.js [hunk 2]
```diff
@@ -89,3 +80,43 @@ exports.addBinToUserPath = () => {
...
+            ? `File already exists: ${cliLinkPath}`
+            : `Symlink creation failed: ${err.code}`;
+
+        //eslint-disable-next-line no-console
+        console.error(err);
+        logNotify(withNotification, 'Hyper CLI installation failed', error);
+      });
+  } else {
+    withNotification &&
+      notify('Hyper CLI instalation', 'Command is added in PATH only at package installation. Please reinstall.');
```
Comment (albinekb) [inline app/utils/cli-install.js]: Should we open an issue about this?

Comment (chabou) [reply app/utils/cli-install.js]: about what? Can you detail your thoughts?

**Comment by Stanzilla** [comment]
Do we not want to do https://github.com/zeit/hyper/issues/2823#issuecomment-381440198 then?

**Comment by chabou** [comment]
@Stanzilla this is another feature (comment added there).

**Pull request merged**